website/docs/troubleshooting/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 1
+---
+
+# Troubleshooting overview
+
+If you experience any issues with FerretDB, this troubleshooting guide will help you resolve them.
+
+## Connectivity issues
+
+Do you have trouble setting up or connecting to FerretDB?
+Find solutions to common connectivity issues below.
+
+### Error initializing PostgreSQL with the DocumentDB extension in Docker
+
+If you get an error when initializing PostgreSQL with the DocumentDB extension in Docker, it may be due to an existing PostgreSQL data directory or volume.
+This error occurs because the previous PostgreSQL data directory was created without the DocumentDB extension.
```

Comment:
Reading above sentence, it's not so clear to me what is the cause.

The reason is PostgreSQL Docker image skips initialization and extension installation if data volume or data directory is already present. So DocumentDB extension is not installed.